WebHostLib/static/assets/player-settings.js

```diff
@@ -145,8 +145,13 @@ const buildOptionsTable = (settings, romOpts = false) => {
         range.setAttribute('min', settings[setting].min);
         range.setAttribute('max', settings[setting].max);
         range.value = currentSettings[gameName][setting];
+        const settingMax = settings[setting].max;
+        const specialMax = settings[setting].specialMax;
         range.addEventListener('change', (event) => {
-          document.getElementById(`${setting}-value`).innerText = event.target.value;
+          const v = event.target.value;
+          // if at max and have specialMax, use that, else use value
+          const text = ((v == settingMax) && specialMax) || v;
```

Comment:
This evaluation uses a quirk of JS to perform assignment, and should be avoided.
Also, comparison should always use `===`.

Reply:
`===` doesn't work because the value from html is a string and settingMax is a number.

Reply:
In that case, I'd recommend using `parseInt(event.target.value, 10)`.

Referencing an above comment, if you wanted to do something like: `const val = parseInt(event.target.value, 10);` and reference `val` more than once later, that would be proper.

As an aside, single letter variable names should also generally be avoided unless they are looping variables.